src/main/java/org/truffleruby/language/RubyGuards.java

```diff
@@ -197,7 +197,7 @@ public static boolean isRubyPointer(DynamicObject value) {
         return Layouts.POINTER.isPointer(value);
     }
 
-    public static boolean isRubiniusByteArray(DynamicObject value) {
+    public static boolean isSupportByteArray(DynamicObject value) {
```

Comment:
As part of the rename, it'd be nice if this were renamed to `isTruffleByteArray`. Just something to note when you change the namespace again.

Reply:
Will do.